packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -4621,6 +4786,105 @@ describe('plugin-meetings', () => {
         });
       });
 
+      describe('#injectMeetingInfo', () => {
+        const FAKE_PASSWORD = '123456';
+        const FAKE_CAPTCHA_CODE = '654321';
+        const FAKE_DESTINATION = 'something@somecompany.com';
+        const FAKE_TYPE = _SIP_URI_;
+        const FAKE_INSTALLED_ORG_ID = '123456';
+        const FAKE_EXTRA_PARAMS = {
+          mtid: 'm9fe0afd8c435e892afcce9ea25b97046',
+          joinTXId: 'TSmrX61wNF',
+        };
+
+        const FAKE_SDK_CAPTCHA_INFO = {};
+        const FAKE_MEETING_INFO = {
+          conversationUrl: 'some_convo_url',
+          locusUrl: 'some_locus_url',
+          sipUrl: 'some_sip_url', // or sipMeetingUri
+          meetingNumber: '123456', // this.config.experimental.enableUnifiedMeetings
+          hostId: 'some_host_id', // this.owner;
+        };
+        const FAKE_MEETING_INFO_LOOKUP_URL = 'meetingLookupUrl';
+
+        it('calls meetingInfoProvider with all the right parameters and parses the result', async () => {
+          meeting.requiredCaptcha = FAKE_SDK_CAPTCHA_INFO;
```

Comment:
suggestion: This is a little confusing, since for the inject case, a captcha is never needed - it is initialised as `null` and is only set to something non-`null` when doing an actual fetch. I realise the reason for this is because you also provide a captcha code, but again I don't think this should be provided generally for inject info. I think it may be simpler to just not set / provide anything captcha (or password) related for this first test.